app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
   },
   "yourSigRequested": {
     "message": "Tu firma ya fue solicitada"
+  },
+  "contractInteraction": {
+    "message": "Interacción del Contrato"
+  },
+  "providerRequest": {
+    "message": "$1 quisiera conectar con tu cuenta"
+  },
+  "acceleratingATransaction": {
+    "message": "* Agilizar a una transacción al usar un precio de gas más alto aumenta las probabilidades que que la red lo procese más rápidamente, pero eso no siempre está garantizado."
+  },
+  "accessingYourCamera": {
+    "message": "Accediendo a tu cámara..."
+  },
+  "accountOptions": {
+    "message": "Opciones de la Cuenta"
+  },
+  "accountSelectionRequired": {
+    "message": "¡Necesitas elegir una cuenta!"
+  },
+  "activityLog": {
+    "message": "registro de actividades"
+  },
+  "advancedOptions": {
+    "message": "Opciones Avanzadas"
+  },
+  "addSuggestedTokens": {
+    "message": "Agregar tokens propuestos"
+  },
+  "addAcquiredTokens": {
+    "message": "Agregar los tokens que has adquirido usando MetaMask"
+  },
+  "advanced": {
+    "message": "Avanzado"
+  },
+  "amountPlusTxFee": {
+    "message": "Cantidad + Comisión por transacción"
+  },
+  "attemptToCancel": {
+    "message": "¿Intentar cancelar?"
+  },
+  "attemptToCancelDescription": {
+    "message": "El intentar cancelar la transacción no garantiza la cancelación de la misma. Si el intento de cancelación tiene éxito, se le cobrará la comisión de transacción indicada arriba."
+  },
+  "average": {
+    "message": "promedio"
+  },
+  "basic": {
+    "message": "Básico"
+  },
+  "browserNotSupported": {
+    "message": "Tu navegador no está admitido..."
+  },
+  "bytes": {
+    "message": "Bytes"
+  },
+  "cancelAttempt": {
+    "message": "Intento de Cancelar"
+  },
+  "cancellationGasFee": {
+    "message": "Comisión de Gas por Cancelación"
+  },
+  "cancelled": {
+    "message": "Cancelado"
+  },
+  "cancelN": {
+    "message": "Cancelar a todas transacciones de $1"
+  },
+  "clickToAdd": {
+    "message": "Hazle clic en $1 para agregarles a su cuenta"
+  },
+  "close": {
+    "message": "Cerrar"
+  },
+  "chromeRequiredForHardwareWallets": {
+    "message": "Hay que usar MetaMask en Google Chrome para poder conectarse con tu Monedero Físico."
+  },
+  "confirmationTime": {
+    "message": "Tiempo de confirmación (seg)"
+  },
+  "connectHardwareWallet": {
+    "message": "Conectar Monedero Físico"
+  },
+  "connect": {
+    "message": "Conectar"
+  },
+  "connectRequest": {
+    "message": "Petición para Conectar"
+  },
+  "connecting": {
+    "message": "Conectándose..."
+  },
+  "connectingTo": {
+    "message": "Conectánodse a $1"
+  },
+  "connectToLedger": {
+    "message": "Conectarse al Ledger"
+  },
+  "connectToTrezor": {
+    "message": "Conectarse al Trezor"
```

Comment:
This and "Conectarse al Ledger" sounds odd to me. I'd use "Conectarse a Ledger"  but this is probably a different Spanish speaker bias

Reply:
Well, I understand what you're saying. I think my rationale behind including the article with the proposition was because it's a specific thing we're talking about, we're connecting to "the Ledger", the one you have. But I see what you're saying, it's not strictly necessary. I'll mull it over, might change it.